comment_iowriter.go

```diff
@@ -118,6 +124,8 @@ func (cw *RDJSONCommentWriter) Post(_ context.Context, c *Comment) error {
 	return nil
 }
 
+func (*RDJSONCommentWriter) ShouldPrependGitRelDir() bool { return false }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method RDJSONCommentWriter.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChAxM2NjYzU2NTNlZDExOGFmEg9nb2xpbnQtY2lyY2xlY2k= -->